### PR TITLE
Defold

### PR DESCRIPTION
**Pull request by 8bitskull**
Add Defold

### CODE REVIEW

File: descriptions/Engine.Defold.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Defold is a completely free to use game engine for development of console, desktop, mobile and web games. There are no up-front costs, no licensing fees and no royalties. The source code is made available on GitHub with a developer-friendly license. The Defold editor runs on Windows, Linux and macOS and includes a code editor, debugger, profiler and advanced scene and UI editors. Game logic is written in Lua with the option to use native code to extend the engine with additional functionality. Defold is used by a growing number of developers to create commercial hits as well as games for game jams and in schools to teach game development. Defold is known for its ease of use and it is praised for its technical documentation and friendly community of developers.
```
Comment (xPaw) [inline descriptions/Engine.Defold.md:1]: Can you make this a bit shorter than turn `Defold` into a link

**Comment by mimattr** [comment]
@8bitskull The `game.dmanifest` file may be a little too generic and may benefit from referencing the archive data and index files instead (`game.arcd` and `game.arci` respectively):

`(?:^|/)game\.arc(?:d|i)$`

_**Note:** Defold's `game.*` files are consistent across all supported platforms (e.g. Android, iOS, Linux, macOS, Windows)_

**Comment by mimattr** [comment]
@8bitskull The `game.dmanifest` file may be a little too generic and would probably benefit from referencing the archive data and index files instead (`game.arcd` and `game.arci` respectively):

`(?:^|/)game\.arc(?:d|i)$`

_**Note:** Defold's `game.*` files are consistent across all supported platforms (e.g. Android, iOS, Linux, macOS, Windows)_

**Comment by xPaw** [comment]
game.dmanifest matches the exact same games as game.arcd and game.arci, so any of the three is fine.

**Comment by mimattr** [comment]
> game.dmanifest matches the exact same games as game.arcd and game.arci, so any of the three is fine.

@xPaw Good to know though the concern was referencing files with common/generic terms like `*manifest` rather than more distinct/unique names (e.g. `.arcd`, `.arci`), helping reduce potential conflicts with other applications moving forward.

**Review by xPaw — APPROVED** [review]

**Pull request merged**